src/ExpertExtender/ExpertExtender.Preview.js

```diff
@@ -11,17 +11,15 @@
 	 * @constructor
 	 *
 	 * @param {Function} getUpstreamValue A getter for the current formatted upstream value
+	 * @param {util.MessageProvider} messageProvider
 	 */
-	ExpertExtender.Preview = function( getUpstreamValue ) {
+	ExpertExtender.Preview = function( getUpstreamValue, messageProvider ) {
 		this._getUpstreamValue = getUpstreamValue;
 
-		var messageProvider = null;
-		if( mw && mw.msg ) {
-			messageProvider = new MessageProvider( {
-				messageGetter: mw.msg,
-				prefix: 'valueview-preview-'
-			} );
-		}
+		var messageProvider = new util.PrefixingMessageProvider(
```

Comment:
Is it ok to access `util` here without it being mentioned in the list of dependencies (in the `( jQuery, jQuery.valueview.ExpertExtender )` line below)?